phpstan-baseline.neon

```diff
@@ -0,0 +1,6 @@
+parameters:
+    ignoreErrors:
+        - # NEXT_MAJOR: Fix this one in SonataBlockBundle 4.x
```

Comment:
Can we agree about how we plan to solve it ?

Changing array to iterable in blockBundle ?
Changing to arraycollection in blockbundle ? 
Using array here ?

Reply:
it can be solved with iterable , I tried it already, but cant do on 3.x (I didnt tried on 4.x tho)

Reply:
Currently 
https://github.com/sonata-project/SonataBlockBundle/blob/4.x/src/Model/BlockInterface.php#L139-L144
is typehinted to `array`.

So it changing the typehint will be mandatory to use SonataBlockBundle indeed.

IMHO you can already add getChildren() and setChildren in the PageBlockInterface with phpdoc about Collection.

Reply:
setChildren is not a method on BlockInterface (and it is not used on for PageBlockInterface

for getChildren, is it needed? for the moment it does not break anything.

Reply:
I think the best would be to do the PR on BlockBundle now.

Does this https://github.com/sonata-project/SonataBlockBundle/pull/1087 seems ok to you ?

Reply:
It seems ok (i have left a comment) but can't be tested with this PR because here we are still using block bundle 3